cmd/cli/main.go

```diff
@@ -14,6 +14,160 @@
 
 package main
 
-func main() {
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"os"
+	"path"
+
+	"github.com/spf13/cobra"
+
+	"antrea.io/resource-auditing/pkg/types"
+)
+
+// tag flags
+var tagAuthor, tagEmail string
+
+// rollback flags
+var rollbackTag, rollbackSHA string
+
+// shared flags
+var serverAddr string
+
+var commandName = path.Base(os.Args[0])
+
+var rootCmd = &cobra.Command{
+	Use:  commandName,
+	Long: commandName + " is the command line tool for managing the auditing resource repository",
+}
+
+var tagCmd = &cobra.Command{
+	Use:   "tag create tag_name commit_sha [-a author] [-e email]\n   or: tag delete tag_name",
```

Comment:
Shouldn't `Use` be `tag` (the command name) and this help message be for `Long`?

Reply:
According to cobra doc, `Use` is for the one-line usage message: https://pkg.go.dev/github.com/spf13/cobra#Command

Reply:
I think I'm just trying to understand how cobra knows that `auditctl tag` is handled by `tagCmd`, while `auditctl rollback` is handled by `rollbackCmd`? I may be missing something obvious...

Reply:
from playing with the field, it seems like cobra just parses the first word of the usage message to determine the command to run